modules/social_features/social_profile/modules/social_profile_fields/src/ContentTranslationDefaultsConfigOverride.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\social_profile_fields;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorableConfigBase;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+
+/**
+ * Provides content translation for the Social Profile Fields module.
+ *
+ * @package Drupal\social_profile_fields
+ */
+class ContentTranslationDefaultsConfigOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * The module handler.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * ContentTranslationDefaultsConfigOverride constructor.
+   *
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler service.
+   */
+  public function __construct(ModuleHandlerInterface $module_handler) {
+    $this->moduleHandler = $module_handler;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names): array {
+    $overrides = [];
+
+    // If the module "social_content_translation" is enabled let make
+    // translations enabled for content provided by the module by default.
+    $is_content_translations_enabled = $this->moduleHandler->moduleExists('social_content_translation');
+
+    if (!$is_content_translations_enabled) {
```

Comment:
Remove variable `$is_content_translations_enabled` and use `$this->moduleHandler->moduleExists('social_content_translation')` directly in `if()` since that variable is using only once.

```suggestion
    if (!$this->moduleHandler->moduleExists('social_content_translation')) {
```